components/button/button-move.js

```diff
@@ -104,7 +103,7 @@ class ButtonMove extends ThemeMixin(FocusMixin(RtlMixin(LitElement))) {
 				:host([theme="dark"]) {
 					--d2l-button-move-background-color-focus: #000000;
 					--d2l-button-move-icon-background-color-hover: rgba(51, 53, 54, 0.9); /* tungsten @70% @90% */
-					--d2l-button-move-box-shadow-focus: 0 0 0 2px black, 0 0 0 4px var(--d2l-color-celestine-plus-1);
+					--d2l-button-focus-color: var(--d2l-color-celestine-plus-1);
```

Comment:
For dark mode, we don't need a black inner ring anymore -- it's assumed that the background is dark.